### ISSUE TITLE
Several warnings are displayed

### ISSUE THREAD

**Issue by femtotrader** [MommaWatasu/OteraEngine.jl#40]
Hello,

I can live with that but when running tests

```
WARNING: Method definition input(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition greet(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition say_twice(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition greet(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition say_twice(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition greet(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition say_twice(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition input(Any, Any, Any, Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition greet(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition say_twice(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition greet(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition say_twice(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition greet(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition say_twice(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition greet(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition say_twice(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition greet(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
WARNING: Method definition say_twice(Any) in module OteraEngine at none:1 overwritten on the same line (check for duplicate calls to `include`).
```

is a lot of warnings.

Best

**Comment by MommaWatasu** [comment]
This is known issue and I've been working on this. But currently it just throws warning and causes no problem.
So I'll fix this in someday but not right now.

**Comment by MommaWatasu** [comment]
This problem is solved in v0.6.0 and the usage of filter and macro is changed.
Check it.

**Comment by femtotrader** [comment]
Thanks @MommaWatasu
